packages/integrations/cloudflare/src/index.ts

```diff
@@ -229,10 +249,17 @@ export default function createIntegration(args?: Options): AstroIntegration {
 				_config = config;
 				finalBuildOutput = buildOutput;
 
+				let customWorkerEntryPoint: URL | undefined;
+				if (args?.workerEntryPoint?.path && typeof args.workerEntryPoint.path === 'string') {
+					customWorkerEntryPoint = new URL(args.workerEntryPoint.path, config.root);
```

Comment:
Would it be possible to pass the value through directly? `serverEntrypoint` can be a string, which allows it to use entrypoints exported by modules and not just local ones.

Reply:
Would Astro take that path relative to the project root?

Reply:
@ascorbic I tried to be smart about this, but I'm not fully sure I get it. How would I be able to tell if it is a file path or a package handler? Starting with `./` would be a good identification, but `src/worker.ts`, should also work. And that could also be an entrypoint for a package called `src`, right?